internal/scripts/docker/Dockerfile

```
@@ -1,30 +1,34 @@
 ARG MET_BASE_REPO=met-base-metviewer
-ARG MET_BASE_TAG=v3.3
+ARG MET_BASE_TAG=v3.3.1
 
 FROM dtcenter/${MET_BASE_REPO}:${MET_BASE_TAG}
-MAINTAINER John Halley Gotway <johnhg@ucar.edu>
+LABEL org.opencontainers.image.authors="John Halley Gotway <johnhg@ucar.edu>"
+
 
 #
 # This Dockerfile checks out METviewer and its dependencies from GitHub and builds
 # the specified branch or tag. Use the develop branches for dependencies by default
 # but override with "--build-arg".
 #
 
-ARG METVIEWER_GIT_NAME
-ARG METPLOTPY_GIT_NAME=develop
-ARG METCALCPY_GIT_NAME=develop
-ARG METDATAIO_GIT_NAME=develop
+ARG SOURCE_BRANCH=feature_585_update_jars
```

Sure seems like this reference should be changed. Perhaps to `main_v6.0`?
```suggestion
ARG SOURCE_BRANCH=main_v6.0
```